packages/codegen/src/codegen/__tests__/generate-editor.expected.ts

```diff
@@ -0,0 +1,137 @@
+export const EXPECTED_INDEX_CONTENT = `import type { EditorModule } from 'document-model';
+import type { EditorDocument } from "./editor.js";
+import Editor from './editor.js';
+
+export const module: EditorModule<EditorDocument> = {
+    Component: Editor,
+    documentTypes: ["powerhouse/document-model", ],
+    config: {
+        id: 'test-document-model-editor',
+        disableExternalControls: true,
+        documentToolbarEnabled: true,
+        showSwitchboardLink: true,
+    },
+};
+
+export default module;`;
+
+const EXPECTED_EDITOR_METHOD = `export default function Editor(props: IProps) {
```

Comment:
I think we should start moving away from these default exports. what is the benefit of doing this instead of a named export? mixing named and default exports makes the bundlers and build tools more likely to get confused, and if you want to do an alias then you have to do a weird export { default as Something }